iotdb-client/jdbc/src/main/java/org/apache/iotdb/jdbc/IoTDBResultMetadata.java

```diff
@@ -252,7 +278,13 @@ public int getPrecision(int column) throws SQLException {
       case DOUBLE:
         return 308;
       case TEXT:
+      case BLOB:
+      case STRING:
         return Integer.MAX_VALUE;
+      case TIMESTAMP:
```

Comment:
This case doesn't seem to make it to the end.
And the result above is 13, here it is 3